changelogs/unreleased/downgrade.md

```diff
@@ -1,4 +1,4 @@
 ## feature/box
 
-* Added support for downgrading system spaces to state compatible with older
-  releases of Tarantool (gh-7718).
+* Added the support for downgrading system spaces to states compatible with
+  older Tarantool versions (gh-7718).
```

Comment:
Plural here is a bit confusing, as for me. Maybe `state` at all. Just `downgrading to make compatible`? Don't know, but I hope that there is a better way to describe it.